setup.rb

```diff
@@ -136,16 +136,9 @@ def system_with_debug(*args)
 options[:os] ||= detect_os
 
 def bootstrap_epel(release)
-  epel = "[bootstrap-epel]\n" \
-          "name=Bootstrap EPEL\n" \
-          "failovermethod=priority\n" \
-          "enabled=0\n" \
-          "gpgcheck=0\n" \
-          "mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?repo=epel-#{release}&arch=$basearch"
-
-  File.open("/etc/yum.repos.d/bootstrap-epel.repo", 'w') { |file| file.write(epel) }
-  system('yum --enablerepo=bootstrap-epel -y install epel-release')
-  system('rm -f /etc/yum.repos.d/bootstrap-epel.repo')
+  unless system("ls /etc/yum.repos.d/epel*")
```

Comment:
I believe this is to account for the fact that sometimes people are setting up epel out of band of the release RPMs such as the Vagrant box we use currently for EL7, the newer version of it lays down a custom repository file instead of installing the release RPM.